testsuite/features/profiles/Makefile

```diff
@@ -4,13 +4,22 @@
 TEMPLATE_DOCKER_DIR = Docker
 TEMPLATE_KIWI_DIR   = Kiwi
 
+REPO = download.suse.de/ibs
```

Comment:
```suggestion
DEFAULT_REPO = download.suse.de/ibs
```

Reply:
Just this is not enough. But I renamed REPO to DEFAULT_REPO